### PR TITLE
[DOC] Updates to schema reference for Kafka config

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**

Updates to the Schema reference appendix in the _Using Guide_.
Part of restructure of the Kafka configuration section

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: documentation/api/io.strimzi.api.kafka.model.EntityTopicOperatorSpec.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+[id='property-topic-operator-logging-{context}']
+=== `logging`
+
+The Topic Operator has a configurable logger:
+
+* `rootLogger.level`
+
+The Topic Operator uses the Apache `log4j2` logger implementation.
+
+Use the `logging` property in the `Kafka` resource to configure loggers and logger levels.
```
Comment (ppatierno) [inline documentation/api/io.strimzi.api.kafka.model.EntityTopicOperatorSpec.adoc:12]: we could highlight that it's needed to be done under the `entityOperator.topicOperator` field of the `Kafka` resource.

File: documentation/api/io.strimzi.api.kafka.model.EntityUserOperatorSpec.adoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+[id='property-user-operator-logging-{context}']
+=== `logging`
+
+The User Operator has a configurable logger:
+
+* `rootLogger.level`
+
+The User Operator uses the Apache `log4j2` logger implementation.
+
+Use the `logging` property in the `Kafka` resource to configure loggers and logger levels.
```
Comment (ppatierno) [inline documentation/api/io.strimzi.api.kafka.model.EntityUserOperatorSpec.adoc:12]: Ditto as above, highlighting the field `entityOperator.userOperator`

**Review by ppatierno — COMMENTED** [review]
LGTM. I left just a couple of comments.

**Review by scholzj — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]
I forgot to approve after raising the two nits.

**Review by laidan6000 — APPROVED** [review]
Looks good to me.

**Pull request merged**